README.md

```diff
@@ -71,21 +71,25 @@ Trying to use an unknown key or wrong format arguments is a compile-time error.
 You can use environment variables to provide a different locale at compile time:
 
 ```rust
-use ctl10n;
 use std::env;
+use std::path::Path;
+
+use ctl10n;
+
+const LOCALES: &[&str] = &["de", "en"];
 
 fn main() {
-    println!("cargo:rerun-if-changed=build.rs");
-    println!("cargo:rerun-if-changed=locales/*.toml");
-    if let Err(err) = ctl10n::convert_strings_file(
-        format!(
-            "locales/{}.toml",
-            &env::var("LOCALE").unwrap_or("en".to_string())
-        ),
-        "strings.rs",
-    ) {
-        panic!("{}", err);
+    for locale in LOCALES {
+        println!("cargo:rerun-if-changed=locales/{}.toml", locale);
     }
+    println!("cargo:rerun-if-env-changed=LOCALE");
+    let locale_file = format!(
+        "locales/{}.toml",
+        &env::var("LOCALE").unwrap_or("en".to_string())
+    );
+    let out_file = Path::new(&env::var("OUT_DIR").unwrap()).join("strings.rs");
+    let _ignore_error = std::fs::remove_file(&out_file);
```

Comment:
I'm not sure that ignoring error is a good thing to do.

Reply:
> Please, change documentation in `src/lib.rs` and then regenerate README.md using `cargo-readme`.

Ah, forgot about that. Again. My bad!

> I'm not sure that ignoring error is a good thing to do.

The only errors here are:
* No such file or directory
* Path is a directory
* The user lacks permissions to remove the file

We want to ignore the first error and the other two will cause the macro file creation to fail anyway. Also we have no guarantee that this actually removes the file properly.

Reply:
Ok, but probably it should be a comment in code.